biztalk/core/office365-mail-adapter.md

```diff
@@ -63,18 +63,22 @@ The Office 365 Outlook Email Adapter allows you to send and receive mails from y
 
 1. The default properties may also be updated using promoted properties:
 
-|Property|Promoted property|
-|---|---|
-| **To** | OfficeMail.To |
-| **CC** | OfficeMail.CC |
-| **Subject** | OfficeMail.Subject |
-| **Importance** | OfficeMail.Importance |
+    Property|Promoted property|
+    |---|---|
+    | **To** | OfficeMail&#46;To |

```

Comment:
can you please revert the ascii changes: `&#46;`, `&nbsp;`

Reply:
if I leave the ., it shows hyperlinked in the UI. I don't want a hyperlink.

Reply:
worked fine for me. Subject and Importance have . , how is it working there?

Reply:
Fixed.